project-memberships-add-associations-to-model

From: Bryan Larsen <bryan@larsen.st>

Next, add the associations to the model:

SHOW_PATCH
---
 app/models/project_membership.rb |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/app/models/project_membership.rb b/app/models/project_membership.rb
index 982242c..65f0b8b 100644
--- a/app/models/project_membership.rb
+++ b/app/models/project_membership.rb
@@ -7,6 +7,9 @@ class ProjectMembership < ActiveRecord::Base
   end
   attr_accessible 
 
+  belongs_to :project, :inverse_of => :memberships
+  belongs_to :user, :inverse_of => :project_memberships
+
   # --- Permissions --- #
 
   def create_permitted?
